src/newgrf.cpp

```diff
@@ -2405,10 +2405,10 @@ static ChangeInfoResult TownHouseChangeInfo(uint hid, int numinfo, int prop, Byt
 					housespec->grf_prop.local_id = hid + i;
 					housespec->grf_prop.subst_id = subs_id;
 					housespec->grf_prop.grffile = _cur.grffile;
-					housespec->random_colour[0] = 0x04;  // those 4 random colours are the base colour
-					housespec->random_colour[1] = 0x08;  // for all new houses
-					housespec->random_colour[2] = 0x0C;  // they stand for red, blue, orange and green
-					housespec->random_colour[3] = 0x06;
+					housespec->random_colour[0] = COLOUR_RED; // these 4 random colours are the base colour
```

Comment:
Not your doing, but an inline comment wrapping over two lines and trying to describe four lines is very creative!

Maybe fix that. 😉